_posts/2019-02-28-into-the-darkness-of-viewmodel-lists-composition.md

```diff
@@ -66,18 +68,18 @@ The Marketing `RequestHandler` publishes an in-memory event to notify that some
 
 ![1551362519284](/img/posts/into-the-darkness-of-viewmodel-lists-composition/1551362519284.png){:class="img-responsive"}
 
-Once all components have received the event they go to their respective backends to retrieve all the information related to all the loaded products, in 1 single roundtrip. 
+Once all components have received the event they go to their respective backends to retrieve the information related to all the loaded products, in a single roundtrip. 
```

Comment:
_all_ is mentioned three times in this and the next paragraph.

Reply:
👍🏼